subcommittee/coding-guidelines/initiatives/safety-critical-rust-coding-guidelines/src/coding-guidelines/types-and-traits.rst

```diff
@@ -0,0 +1,54 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Coding Guidelines Subcommittee Contributors
+
+.. default-domain:: coding-guidelines
+
+Types and Traits
+================
+
+.. guideline:: Avoid Implicit Integer Wrapping
```

Comment:
Avoid doesn't sound like a requirement but the MUST below does.  Follow something like https://datatracker.ietf.org/doc/html/rfc2119 to define the meaning of terms you use in the coding standard.  Also follow the update from https://datatracker.ietf.org/doc/html/rfc8174

Reply:
There may also be some cases where you want modulo behavior so you need an exception to this rule and someway for the developer to indicate that they want modulo behavior to suppress a tool diagnostic.